src/extensions/react-flow/nodes/NodeContent.tsx

```diff
@@ -404,5 +413,5 @@ export function NodeContent<CONTENT_PROPS = any>({
         </Resizable>
     );
 
-    return onNodeResize ? resizableNode() : nodeContent;
+    return (!!onNodeResize === true && minimalShape === "none") ? resizableNode() : nodeContent;
```

Comment:
`!!onNodeResize === true` can be reduced to `!!onNodeResize` or just `onNodeResize`.